buddyboss/buddyboss-platform
# New Invitations API Coming in BuddyPress 5.0

### All Submissions:






### Changes proposed in this Pull Request:
In BuddyPress 5.0 they edited code related to Group Memberships and Invites, to use the API. Platform does not have this code. The API is expecting those updated functions and failing.

https://github.com/buddyboss/buddyboss-platform-api/issues/2



Fixes #517.

### How to test the changes in this Pull Request:

1. Check the group invites works.
2. Check the membership request within group work.
3. Check emails with all the invitations functionality with groups.

### Other information:
